From 9b52726df50cc9b4fdfce2ed1dc114311dc60a93 Mon Sep 17 00:00:00 2001
From: Paul Eggert <eggert@cs.ucla.edu>
Date: Sun, 8 Nov 2015 09:36:14 -0800
Subject: * src/unexelf.c (NEW_PROGRAM_H): Remove unused macro (Bug#20614).

Emacs should build on ppc64el.  A problem with the bss has been fixed.

This upstream patch has been added [9/10]:

  * src/unexelf.c (NEW_PROGRAM_H): Remove unused macro (Bug#20614).

Origin: upstream, commit: 2ce0c0674eba9179ba57c537e387bc3f7b0e5b63
Bug: http://debbugs.gnu.org/20614
Bug-Debian: http://bugs.debian.org/808347
Added-by: Rob Browning <rlb@defaultvalue.org>
---
 src/unexelf.c |    4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

Index: emacs-24.5/src/unexelf.c
===================================================================
--- emacs-24.5.orig/src/unexelf.c
+++ emacs-24.5/src/unexelf.c
@@ -208,8 +208,6 @@ entry_address (void *section_h, ptrdiff_
   (*(ElfW (Shdr) *) entry_address (new_section_h, n, new_file_h->e_shentsize))
 #define OLD_PROGRAM_H(n) \
   (*(ElfW (Phdr) *) entry_address (old_program_h, n, old_file_h->e_phentsize))
-#define NEW_PROGRAM_H(n) \
-  (*(ElfW (Phdr) *) entry_address (new_program_h, n, new_file_h->e_phentsize))
 
 typedef unsigned char byte;
 
@@ -250,7 +248,7 @@ unexec (const char *new_name, const char
   ElfW (Phdr) *old_bss_seg, *new_bss_seg;
   ElfW (Addr) old_bss_addr, new_bss_addr;
   ElfW (Word) old_bss_size, new_data2_size;
-  ElfW (Off)  old_bss_offset, new_data2_offset;
+  ElfW (Off) old_bss_offset, new_data2_offset;
 
   ptrdiff_t n;
   ptrdiff_t old_bss_index;
